demos/mcp_server_file_io/file_io_server.py

```
@@ -0,0 +1,326 @@
+import datetime
+import os
+import shutil
```

Ruff (0.11.9)

`shutil` imported but unused

Remove unused import: `shutil`

(F401)